settings/development/PandoraSettings_Streaming_Neutrino_DUNEFD.xml

```diff
@@ -0,0 +1,640 @@
+            <algorithm type = "LArBranchSplitting" />
+            <algorithm type = "LArKinkSplitting" />
+            <algorithm type = "LArTrackConsolidation">
+                <algorithm type = "LArSimpleClusterCreation" description = "ClusterRebuilding"/>
+            </algorithm>
+            <algorithm type = "LArHitWidthClusterMerging"/>
+        </AlgorithmsShowerClustersV>
+    </algorithm>
+
+    <algorithm type = "LArClusteringParent">
+        <algorithm type = "LArTrackClusterCreation" description = "ClusterFormation"/>
+        <InputCaloHitListName>CaloHitListW</InputCaloHitListName>
+        <ClusterListName>ClustersW</ClusterListName>
+        <ReplaceCurrentCaloHitList>true</ReplaceCurrentCaloHitList>
+        <ReplaceCurrentClusterList>true</ReplaceCurrentClusterList>
+    </algorithm>
+    <algorithm type = "LArLayerSplitting"/>
+    <algorithm type = "LArDLTrackShowerStreamSelection">
+        <TrackListName>TrackClustersW</TrackListName>
+        <ShowerListName>ShowerClustersW</ShowerListName>
+    </algorithm>
+
+    <algorithm type = "LArStreaming">
+        <InputListNames>TrackClustersW ShowerClustersW</InputListNames>
+        <OutputListNames>TrackClustersW ShowerClustersW</OutputListNames>
+        <AlgorithmsTrackClustersW>
+            <algorithm type = "LArLongitudinalAssociation" />
+            <algorithm type = "LArTransverseAssociation" />
+            <algorithm type = "LArLongitudinalExtension" />
+            <algorithm type = "LArTransverseExtension" />
+            <algorithm type = "LArCrossGapsAssociation" />
+            <algorithm type = "LArCrossGapsExtension" />
+            <algorithm type = "LArOvershootSplitting" />
+            <algorithm type = "LArBranchSplitting" />
+            <algorithm type = "LArKinkSplitting" />
+            <algorithm type = "LArTrackConsolidation">
+                <algorithm type = "LArSimpleClusterCreation" description = "ClusterRebuilding"/>
+            </algorithm>
+            <algorithm type = "LArHitWidthClusterMerging"/>
+        </AlgorithmsTrackClustersW>
+        <AlgorithmsShowerClustersW>
+            <algorithm type = "LArLongitudinalAssociation" />
+            <algorithm type = "LArTransverseAssociation" />
+            <algorithm type = "LArLongitudinalExtension" />
+            <algorithm type = "LArTransverseExtension" />
+            <algorithm type = "LArCrossGapsAssociation" />
+            <algorithm type = "LArCrossGapsExtension" />
+            <algorithm type = "LArOvershootSplitting" />
+            <algorithm type = "LArBranchSplitting" />
+            <algorithm type = "LArKinkSplitting" />
+            <algorithm type = "LArTrackConsolidation">
+                <algorithm type = "LArSimpleClusterCreation" description = "ClusterRebuilding"/>
+            </algorithm>
+            <algorithm type = "LArHitWidthClusterMerging"/>
+        </AlgorithmsShowerClustersW>
+    </algorithm>
+
```

Comment:
Could potentially use the streaming alg to reduce repetition in the XML file, but at a cost of clarity. Sometimes repetition is better than needing to know how an algorithm calls a named list of child algs.

Reply:
At some point I'd like to update the streaming algorithm to allow specification of the set of views to run over and eliminate the repetition.